tasks/run-zap/task.sh

```diff
@@ -25,5 +26,6 @@ done
 zap-cli shutdown
 
 echo Uploading files...
-python scripts/tools/s3upload.py
-
+# credentials are provided via environment variables
+# http://docs.aws.amazon.com/cli/latest/userguide/cli-chap-getting-started.html#cli-environment
+aws s3 sync results "s3://$S3_BUCKET/results"
```

Comment:
It _shouldn't_ be any different from the behavior before...it simply takes the files passed in via the `results` folder, checks to see if they exist in the S3 bucket, then creates/updates each as necessary.